lib/rubocop/cop/mixin/statement_modifier.rb

```diff
@@ -42,8 +42,9 @@ def modifier_fits_on_single_line?(node)
       def length_in_modifier_form(node, cond)
         keyword = node.loc.keyword
         indentation = keyword.source_line[/^\s*/]
-        line_length("#{indentation}#{node.body.source} #{keyword.source} " \
-                    "#{cond.source}")
+        source = "#{indentation}#{node.body.source} #{keyword.source} #{cond.source}"
+        source = "#{node.parent.loc.name} = #{source}" if node.parent&.lvasgn_type?
```

Comment:
Best use an `else` and build `source` only once

Reply:
@marcandre Done. Had to pull out `condition_source` to overcome `Metrics/AbcSize` offense.

Reply:
Looks great to me 👍